src/lib/types/TableState.ts

```diff
@@ -10,7 +10,8 @@ type FilterModel = { [k: string]: any };
 /**
  * The table sorting state
  */
-interface TableState<TData, TFilter extends FilterModel> extends Pick<CoreOptions<TData>["state"], "pagination" | "rowSelection"> {
+interface TableState<TData, TFilter extends FilterModel>
+  extends Pick<CoreOptions<TData>["state"], "pagination" | "rowSelection" | "expanded"> {
```

Comment:
@skuengneo I would not take the definition from tankstack as it's not strongly typed.

You can declare the same type on our side but it should like instead of 

`export type ExpandedState = true | Record<string, boolean>
`

like

`export type ExpandedState <TData>= true | Record<keyof TData, boolean>
`

Reply:
@neotob We need to investigate on this case, probably not so easy to achieve the strongly typed version (the record of string is not the type but the value of the rowID). I think it's ok like this for now :D